charts/super-agent/values.yaml

```diff
@@ -0,0 +1,181 @@
+    # -- The repository where the super-agent has the chart. This values is meant to be changed only on air-gapped 
+    # environments or for development/testing purposes.
+    url: https://helm-charts.newrelic.com
+
+    # -- A reference to a secret with the keys username and password to authenticate to the repository.
+    # @default -- `nil` (no secret reference) 
+    secretRef:
+      # name: name-of-the-secret
+    # -- secret of type `kubernetes.io/tls` with the standard keys `tls.crt`, `tls.key`, and `ca.crt`
+    # @default -- `nil` (no secret reference) 
+    certSecretRef:
+      # name: name-of-the-secret
+
+  # -- Values related to the super agent's Helm chart release.
+  # @default -- See `values.yaml`
+  release:
+    # -- The Helm chart of the super-agent. This values is meant to be changed only on air-gapped environments or for
+    # development/testing purposes.
+    chart: super-agent-deployment
+    # -- The Helm chart of the super-agent. This values is meant to be changed only on air-gapped environments or for
+    # development/testing purposes.
+    #
+    # TODO: Point renovatebot here.
+    version: "0.0.0-beta"
+
+    # -- Set values to the super agent helm release directly from this `values.yaml` file. Refer to https://fluxcd.io/flux/components/helm/helmreleases/#values-overrides
+    # @default -- no values
+    values:
+    # replicaCount: 5
+
+    # -- Set values from a `configMap` or a `secret`. You can see examples and better documentation inside the
+    # `values.yaml` file. Also refer to https://fluxcd.io/flux/components/helm/helmreleases/#values-overrides
+    # @default -- empty
+    valuesFrom:
+
+      # - kind: ConfigMap or Secret
+      #   name: Name of the values referent, in the same namespace as the HelmRelease.
+      #   valuesKey (Optional): The data key where the values.yaml or a specific value can be found. 
+      #                         Defaults to `values.yaml` when omitted.
+      #   targetPath (Optional): The YAML dot notation path at which the value should be merged. When set, the 
+      #                          valuesKey is expected to be a single flat value. Defaults to `nil` when omitted, which
+      #                          results in the values getting merged at the root.
+
+      # Examples:
+      # - kind: ConfigMap
+      #   name: prod-env-values
+      #   valuesKey: values-prod.yaml
+      # - kind: Secret
+      #   name: prod-tls-values
+      #   valuesKey: crt
+      #   targetPath: tls.crt
+
+    # -- Change the behavior of the operator while installing the chart for the first time. This should only be changed
+    # by advanced users that know what they are doing. Exposes the remediations that the operator is going to try
+    # before give up installing the chart in case it hits an error.
+    # @default -- See `values.yaml`
+    install:
+      crds: CreateReplace  # New CRDs are created, existing CRDs are updated (replaced) but not deleted.
```

Comment:
I think we are not creating crds in the SuperAgent chart, I would simplify this removing this option

Reply:
This is not for the superagent but for the things that Flux is going to deploy. If the Super Agent deploys something that needs a CRD (let's say an absurd example like KEDA or the New Relic's operator for alerts) that will need CRDs and will need to upgrade them.

Reply:
I am not following, isn't this being used in the helm-release (charts/super-agent/templates/helm-release.yaml) of the superAgent? Therefore it is going to be useful only if at some point we have CRDs as part of the SA, right?

Reply:
You are completely right. I don't know what I was thinking.

Removed on [#5573b5b](https://github.com/newrelic/helm-charts/pull/1175/commits/5573b5b638dce8efdb24dae685bdaab622774bed)